src/components/CheckoutGrid/styles.js

```diff
@@ -0,0 +1,41 @@
+import styled from 'styled-components'
+
+export const CheckoutGridWrapper = styled.section`
```

Comment:
For convention, the file name that contains the styled-components is `styled.js` and the plain CSS styles is `styles.js`